src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -64,6 +64,7 @@ public RootElement convertToPeers( final Map<String, Map<String, Object>> data,
 			final BufferedImage screenshot ) {
 		final Map<String, WebElementPeer> converted = new HashMap<>();
 		RootElementPeer root = null;
+		final DefaultValuesProvider defaultValuesProvider = new DefaultValuesProvider();
```

Comment:
You really want to load the defaults every time a conversion is triggered? Why not as a static field?

Reply:
Good point. Thanks!